index.js

```diff
@@ -506,12 +542,27 @@ function send (res, type, body) {
   res.end(body, 'utf8')
 }
 
+/**
+ * Convert size to human-readable size
+ */
+
+var SIZES = [ 'B', 'K', 'M', 'G', 'T', 'P', 'E' ]
+function hsize(size) {
```

Comment:
Use a module here instead of adding this code to the module. The `bytes` module is one I know, but you're welcome to use a different one if there is one you like better.

Reply:
That makes me sad. There's so many packages in node that do next to nothing and somehow end up including 100's of megabytes of download. The bytes package is WAY overkill for something this simple.

That said, I'll make the change.

Reply:
(especially considering the recent fiasco with eslint - who knows what percentage of npm was compromised from that alone)

Reply:
It can be a diff package if there is a better one like I said. The other alternative is that this would need tests to cover the conversations (for instance I deleted the "E" from the list and no tests seemed to catch that -- I also changed 1024 to 999 and nothing failed) and it creates more things I have to maintain in the long term, which is harder when it doesn't have tests.

Reply:
> (especially considering the recent fiasco with eslint - who knows what percentage of npm was compromised from that alone)

Ok, but why would it be any different that this module itself cannot be compromised?